src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -897,3 +897,15 @@ def _decode_nonce(backend, nonce):
     get_ext=lambda backend, x, i: backend._lib.OCSP_BASICRESP_get_ext(x, i),
     handlers=_OCSP_BASICRESP_EXTENSION_HANDLERS,
 )
+
+_OCSP_SINGLERESP_EXT_PARSER_NO_SCT = _X509ExtensionParser(
+    ext_count=lambda backend, x: backend._lib.OCSP_SINGLERESP_get_ext_count(x),
+    get_ext=lambda backend, x, i: backend._lib.OCSP_SINGLERESP_get_ext(x, i),
+    handlers=_EXTENSION_HANDLERS_NO_SCT
```

Comment:
What is the set of allowable extensions here? Looking at the RFC it appears that "any CRLEntry extension" from RFC 5280 section 5.3 is allowable + the set defined in RFC 6960. Are there any others? The set of handlers here doesn't look right based on that.